README.md

```diff
@@ -271,6 +271,31 @@ In order to add a new workspace, one should create a new directory under `packag
 
 Suggested way to add a new dependency to an existing workspace is to run `yarn workspace <workspace name> add <package name>` instead of just `yarn add <package name>`. For example, run `yarn workspace cbioportal-frontend add lodash` instead of `yarn add lodash`. Similarly to remove a package run `yarn workspace <workspace name> remove <package name>`.
 
+### Tips for dependency management
```

Comment:
Suggestion: add a subheading titled `Adding a New Workspace` before line 270. You already have the information, but it doesn't stand out.